syscalls.go

```diff
@@ -345,6 +377,10 @@ func wrapMapError(err error) error {
 		return ErrKeyExist
 	}
 
+	if errors.Is(err, unix.ENOTSUPP) {
+		return ErrBatchOpNotSup
```

Comment:
I didn't see this before, this is going to trigger false positives if some other map non-batch command returns ENOTSUPP. Why not just return ErrNotSupported in this case? Seems like you have a specific use case for ErrBatchOpNotSup in mind?

Reply:
I did when we were doing this by map type, but now that we're not your suggestion is correct.